packages/neuron-wallet/src/appTypes/types.ts

```diff
@@ -0,0 +1,63 @@
+// define types in app
+
+export interface Block {
+  header: BlockHeader
+  transactions: Transaction[]
+}
+
+export interface BlockHeader {
+  version: number
+  timestamp: string
+  hash: string
+  parentHash: string
+  number: string
+}
+
+export interface TransactionWithoutHash {
+  version: number
+  deps?: OutPoint[]
+  inputs?: Input[]
+  outputs?: Cell[]
+  timestamp?: string
+  value?: string
+  blockNumber?: string
+  blockHash?: string
+  witnesses?: Witness[]
+  type?: string
+}
+
+export interface Transaction extends TransactionWithoutHash {
+  hash: string
+}
+
+export interface Input {
+  previousOutput: OutPoint
+  args: string[]
+  since?: string
+  capacity?: string | null
+  lockHash?: string | null
+}
+
+export interface Witness {
+  data: string[]
+}
+
+export interface Cell {
+  capacity: string
+  data?: string
+  lock: Script
+  type?: Script | null
+  outPoint?: OutPoint
+  status?: string
+  lockHash?: string
+}
+
+export interface OutPoint {
```

Comment:
Better name it `CellOutPoint` to keep consistent with types in CKB

Reply:
It's still `OutPoint`, and v0.12 will add `CellOutPoint` and update `OutPoint`